consensus/src/runway/mod.rs

```diff
@@ -171,18 +172,65 @@ impl<'a, H: Hasher> RunwayStatus<'a, H> {
     }
 }
 
+fn short_report(rounds_behind: Round, missing_coords: usize) -> String {
+    if rounds_behind <= 2 && missing_coords == 0 {
+        "healthy".to_string()
+    } else if missing_coords == 0 {
+        format!("behind by {rounds_behind} rounds")
+    } else if rounds_behind == 0 {
+        format!("syncing - missing {missing_coords} unit(s)")
+    } else {
+        format!("syncing - missing {missing_coords} unit(s) and behind by {rounds_behind} rounds")
+    }
+}
+
+fn format_missing_coords(c: &[(usize, Round)]) -> String {
+    c.iter()
+        .sorted()
```

Comment:
Maybe use [`sort_by_key`](https://doc.rust-lang.org/stable/std/vec/struct.Vec.html#method.sort_by_key) and use just the first element of the tuple?